content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -30,13 +30,13 @@ you have selected.
 {{< feature-state for_k8s_version="v1.12" state="beta" >}}
 
 ### KMS v2
-* Kubernetes version 1.25.0 or later is required
+* Kubernetes version 1.27.0 or later is required
```

Comment:
For k8s 1.25, this feature is available, right?

Reply:
It is (in alpha). We should do our usual thing and use a feature state shortcode, then add explanatory text about what to do for earlier versions.